### ISSUE TITLE
Doc block braces don't get lexed unless they have space between them

### ISSUE THREAD

**Issue by runarorama** [unisonweb/unison#2148]
`{{{{docAside{{hey}}}}}}` gives a bad error message, but `{{ {{ docAside {{ hey }} }} }}` works fine.

Here's the error:

```
  operator (examples: +, Float./, List.++#xyz)
  resetting state
```

**Comment by kanwren** [comment]
Is this still an issue? That example passes for me, and a bisect said it was fixed in 7745fbba4be701cb6bbae0d26cb19eac90798c43.

**Comment by aryairani** [comment]
@nprindle I didn’t check, but if the example works for you then I would say it’s fixed!
